src/supervisor.adoc

```diff
@@ -730,6 +730,7 @@ characteristics of the U-mode execution environment.
   {bits:  2, name: 'CBIE'},
   {bits:  1, name: 'CBCFE'},
   {bits:  1, name: 'CBZE'},
+  {bits:  1, name: 'SVKT'},
```

Comment:
Since Sekt applies only to RV64 or larger, wouldn't it make sense to put `henvcfg. HUVKT` and `senvcfg.SVKT` at bit offset >= 32, like was done for PMM? This avoids consuming bits in the low word that are irrelevant for RV32.

Reply:
The ARC was not completely confident that we would never define an RV32 variant of this feature, hence locating the bit down here.